common/lib/lockfile.ts

```diff
@@ -0,0 +1,25 @@
+import lockfile from 'lockfile';
+
+export function lockFileAsync( LOCKFILE_PATH: string, options: lockfile.Options ) {
```

Comment:
```suggestion
export function lockFileAsync( lockfilePath: string, options: lockfile.Options ) {
```
Let's not capitalize this, since this is no longer a constant.